README.md

```diff
@@ -11,6 +11,8 @@
 
 Scarpe isn't feature complete with any version of Shoes (yet?). We're initially targeting Shoes Classic.
 
+Scarpe requires [Ruby 3.2](https://www.ruby-lang.org/en/downloads/) or higher! use `rvm` or `rbenv` for version control
```

Comment:
I might say something like "you can install Ruby 3.2 with rvm, rbenv or another version manager or installer." When you say "version control" people tend to think, like, Git.

Reply:
What is the state of rvm these days? I used it for 5-6 years but since changed to `chruby` with `ruby-install`. Is it still going well? I see PRs getting merged

Reply:
I would consider not having this information in the first headline of the README, where we have two lines sharing the highlight of what scarpe is.

Something I would consider is maybe under ## Installation [here](https://github.com/scarpe-team/scarpe/blob/6fe413d38ce919d2f5cd485b36338c4e591ac88e/README.md#installation)

You could have something like:

```md
**Ruby Version**: >= 3.2
```

Reply:
Pretty sure rvm is still going strong. They keep threatening a rewrite for version 2, though it doesn't look like they have it released yet.

Reply:
okay i will change it accordingly